libraries/src/Application/CMSApplication.php

```diff
@@ -900,46 +900,78 @@ public function login($credentials, $options = [])
 
         if ($response->status === Authentication::STATUS_SUCCESS) {
             /*
-             * Validate that the user should be able to login (different to being authenticated).
-             * This permits authentication plugins blocking the user.
-             */
+            * Validate that the user should be able to login (different to being authenticated).
+            * This permits authentication plugins blocking the user.
+            */
```

Comment:
```suggestion
             * Validate that the user should be able to login (different to being authenticated).
             * This permits authentication plugins blocking the user.
             */
```
Please revert this unrelated and wrong doc block code style change.